string label
string pin
geometry_msgs/Pose component_pose
geometry_msgs/Pose screw_pose
geometry_msgs/Pose hole_pose
